components/Player/PlayerMMR.jsx

```diff
@@ -5,15 +5,37 @@ import { connect } from 'react-redux';
 import { player } from '../../reducers';
 import styles from './PlayerHeader.css';
 
+//TODO make a dumb component
 export const PlayerMMR = ({ loading, error, rank, soloRank, mmrEstimate }) => {
   const getPlayerMMR = () => {
     if (error) return <Error />;
     if (loading) return <Spinner />;
     return (
-      <div className={styles.mmr}>
-        <div>Solo: {soloRank}</div>
-        <div>Party: {rank}</div>
-        <div>Estimate: {mmrEstimate.estimate}</div>
+      <div>
+        <span className={"text-info rating"}>
+          <span>
+            <abbr title={"Solo MMR"}>
+              <i className={"fa fa-fw fa-user"}></i>
+            </abbr>
+          </span>
+          <small>{soloRank}</small>
+        </span>
+        <span className={"text-warning rating"}>
+          <span>
+            <abbr title={"Party MMR"}>
+              <i className={"fa fa-fw fa-users"}></i>
+            </abbr>
+          </span>
+          <small>{rank}</small>
+        </span>
+        <span className={"rating"}>
+          <span>
+            <abbr title={"MMR estimate based on available data from peer players. This is an estimate of the population mean MMR of the recent matches played by this user."}>
+              <i className={"fa fa-fw fa-question-circle"}></i>
+            </abbr>
+          </span>
+          <small>{mmrEstimate.estimate}</small>
+        </span>
       </div>
```

Comment:
Yea we should just go with the tooltips idea unless we come up with a better one (we can always refactor). Best to stick with the current solution since what I came up with isn't all that great.